man/portage.5

```diff
@@ -843,15 +843,14 @@ x11\-libs/qt \-mysql
 .TP
 .BR postsync.d/
 This directory is for user supplied postsync hooks to be run once after all
-repositories have been synced.  Each script is called in alphabetic order
+repositories have been synced.  Each script is called in lexical order
 without any arguments.
 .fi
 .TP
 .BR repo.postsync.d/
 This directory is for user supplied postsync hooks to be run once after each
-repository has been synced.  Each script is called in alphabetic order
-with three arguments.
-
+repository has been synced.  Each script is called in lexical order
+with the following three arguments:
```

Comment:
@zmedico - minor tweak here, and dropped the extra line-break. Hopefully this flows better ...